recipes-kernel/linux/5.15/patches/blktap2.patch

```diff
@@ -1057,26 +1054,20 @@ PATCHES
 +	del_gendisk(gd);
 +	gd->private_data = NULL;
 +
-+	cleanup_queue(gd->queue);
++	blk_cleanup_disk(gd);
```

Comment:
With the cleanup_queue removal, blk_cleanup_disk takes the place of blk_cleanup_queue?  Another 5.15 change?

Reply:
Yea, ```blk_cleanup_disk``` does the ```blk_cleanup_queue``` and ```put_disk```(also removed from line 1062) in the same function.